lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -567,13 +595,17 @@ class IdentityCreatorController extends BaseController {
       } else {
         logError("IdentityCreatorController::_insertInlineImage: context is unmounted");
       }
+    } else if (publicAssetController != null) {
+      publicAssetController?.uploadFileToBlob(file);
```

Comment:
If it is already non-null, we can use the `!` operator instead of `?`

```suggestion
      publicAssetController!.uploadFileToBlob(file);
```

Reply:
Done